infra/stack/eks.tf

```diff
@@ -0,0 +1,69 @@
+module "label" {
+  source = "cloudposse/label/null"
+
+  name       = var.label_name
+  attributes = ["cluster"]
+}
+
+locals {
+  # Prior to Kubernetes 1.19, the usage of the specific kubernetes.io/cluster/* resource tags below are required
+  # for EKS and Kubernetes to discover and manage networking resources
+  # https://www.terraform.io/docs/providers/aws/guides/eks-getting-started.html#base-vpc-networking
+  tags = { "kubernetes.io/cluster/${module.label.id}" = "shared" }
+}
+
+module "vpc" {
+  source = "cloudposse/vpc/aws"
+
+  cidr_block = "172.16.0.0/16"
+
+  tags    = local.tags
+  context = module.label.context
+}
+
+module "subnets" {
+  source = "cloudposse/dynamic-subnets/aws"
+
+  availability_zones   = var.availability_zones
+  vpc_id               = module.vpc.vpc_id
+  igw_id               = module.vpc.igw_id
+  cidr_block           = module.vpc.vpc_cidr_block
+  nat_gateway_enabled  = true
+  nat_instance_enabled = false
+
+  tags    = local.tags
+  context = module.label.context
+}
+
+module "eks_cluster" {
+  source = "cloudposse/eks-cluster/aws"
+  region = var.region
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.subnets.public_subnet_ids
+
```

Comment:
`terraform fmt` is helpful to ensure consistent formatting of equal signs, etc. We even have it as a GH action on PRs to ensure it always passes. No need to address on this PR, just an FYI as well

Reply:
Sure, Will try to add that step before every push